### PR TITLE
Add specialIssues to 526 form schema

### PR DESCRIPTION
**Pull request by templeman15**
## Description of change
Add `specialIssues` to form 526 schema. 

## Original issue(s)
[Add specialIssues to the Claims API 526 Schema](https://vajira.max.gov/browse/API-1480)

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb [hunk 1]
```diff
@@ -454,6 +465,17 @@ class Form526ResponseSwagger
                         key :description, 'How the veteran got the disability.'
                         key :example, 'Caused by a service-connected disability\\nLengthy description'
                       end
+
+                      property :specialIssues do
```
Comment (templeman15) [inline modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb:469]: @bastosmichael I'm not positive if `specialIssues` needs to be in `secondaryDisabilities` or not.

File: modules/claims_api/config/schemas/526.json [hunk 2]
```diff
@@ -542,6 +552,9 @@
                 },
                 "serviceRelevance": {
                   "type": "string"
+                },
+                "specialIssues": {
```
Comment (bastosmichael) [inline modules/claims_api/config/schemas/526.json:556]: Why does this exist twice in the schema?

**Review by bastosmichael — CHANGES_REQUESTED** [review]

File: modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb:469]: I do not believe it does, not sure where you got that assumption?

Comment (templeman15) [reply modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb:469]: When reviewing:

https://github.com/department-of-veterans-affairs/vets-json-schema/blob/0f7d93fdcff0139345998d95a80c2cfd1cb65f7a/src/schemas/21-526EZ-allclaims/schema.js#L125

Line 29 `specialIssues` is nested under `disabilitiesBaseDef`. And in turn `disabilitiesBaseDef` is referenced on line 149 and 153 for both `disabilities` and `secondaryDisabilities`.

File: modules/claims_api/config/schemas/526.json (hunk 2 above)
Comment (templeman15) [reply modules/claims_api/config/schemas/526.json:556]: See: 

https://github.com/department-of-veterans-affairs/vets-api/pull/4931#discussion_r491197426

File: modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb:469]: alright it looks like you did your homework, good job.

File: modules/claims_api/config/schemas/526.json (hunk 2 above)
Comment (bastosmichael) [reply modules/claims_api/config/schemas/526.json:556]: Good job.

**Review by bastosmichael — APPROVED** [review]
LGTM, good work on catching it in two different parts of the schema.

**Pull request merged**